v1/python-sdk/tutorials/automl-with-azureml/forecasting-backtest-many-models/auto-ml-forecasting-backtest-many-models.ipynb

```diff
@@ -389,7 +414,14 @@
     "| **train_pipeline_parameters**      | The set of configuration parameters defined in the previous section. |\n",
     "| **run_invocation_timeout**         | Maximum amount of time in seconds that the ``ParallelRunStep`` class is allowed. This is optional but provides customers with greater control on exit criteria. This must be greater than ``experiment_timeout_hours`` by at least 300 seconds. |\n",
     "\n",
-    "Calling this method will create a new aggregated dataset which is generated dynamically on pipeline execution."
+    "Calling this method will create a new aggregated dataset which is generated dynamically on pipeline execution.\n",
+    "\n",
+    "**Note**: Total time taken for the **training step** in the pipeline to complete = $ t/(p * n) * ts $\n",
```

Comment:
Please change the $ t/(p * n) * ts $ to $ \frac{t}{ (p \times n) \times ts} $

It is not clear from the formula how it is written if the $ ts $ term belongs to the numerator or the denominator. In the line above I assumed it belongs to the denominator. In this case, the parentheses are redundant.